src/multupd.jl

```diff
@@ -7,40 +7,48 @@
 #
 
 mutable struct MultUpdate{T}
-    obj::Symbol         # objective :mse or :div
-    maxiter::Int        # maximum number of iterations
-    verbose::Bool       # whether to show procedural information
-    tol::T              # change tolerance upon convergence
-    lambda::T           # regularization coefficient
+    obj::Symbol     # objective :mse or :div
+    maxiter::Int    # maximum number of iterations
+    verbose::Bool   # whether to show procedural information
+    tol::T          # change tolerance upon convergence
+    lambda_w::T     # L1 regularization coefficient for W
+    lambda_h::T     # L1 regularization coefficient for H
 
     function MultUpdate{T}(;obj::Symbol=:mse,
                             maxiter::Integer=100,
                             verbose::Bool=false,
                             tol::Real=cbrt(eps(T)),
-                            lambda::Real=sqrt(eps(T))) where T
```

Comment:
The removal of this keyword argument in the constructor constitutes a breaking change. We should probably at least put in a deprecation for it by including `lambda=nothing` as a keyword argument and emitting a warning if a non-`nothing` value is provided (and set `lambda_w` and `lambda_h` appropriately to accommodate the user's `lambda` value).

Reply:
@ararslan Thank you for your comment. I modified the code.

Reply:
Yes, right. We can name `delta` back to `lambda`, and make something like 
`lambda::Real=sqrt(eps(T)), lambda_w::Real=lambda, lambda_h::Real=lambda` the signature.

Reply:
@mschauer The original `lambda` (default value 1.0e-9) is probably used only to avoid division by zero. After `delta` is named back to the original `lambda`, setting the original `lambda` to a non-small value may adversely affect the accuracy of the approximation X ≈ WH. Therefore, I don't think `delta` should be user-specified.

Reply:
The closer `delta` (the original `lambda`) is to zero, the more ideal the algorithm is. 

cf. https://gist.github.com/tsano430/dc005fd59f89662d777774de84cd5d0d

Reply:
Thank you for your reaction with thumbs up emoji.